{
  "artifacts": [
    "vgg16.medical.json",
    "medical.combo.json"
  ],
  "command": "run",
  "experiment": {
    "base_dir": "/home/steven/Github/keras-training-system/src",
    "dependencies": [
      "Keras==2.2.2",
      "numpy==1.14.5",
      "sacred==0.7.4",
      "tensorflow-gpu==1.10.1"
    ],
    "mainfile": "run.py",
    "name": "medical_vgg16",
    "repositories": [],
    "sources": [
      [
        "evaluate.py",
        "_sources/evaluate_8514da65249e61e3708d0e06c1460320.py"
      ],
      [
        "run.py",
        "_sources/run_0f42912c75b4baa63dbb1497e80ee62e.py"
      ],
      [
        "train.py",
        "_sources/train_eb65b062ac1bb0f54732900d1690704d.py"
      ],
      [
        "utils/logging.py",
        "_sources/logging_18abda31231e258056cc3146fc4c4569.py"
      ],
      [
        "utils/util.py",
        "_sources/util_ef16642f5268bcabcc8ce6036f84aa83.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/sacred/config/captured_function.py\", line 46, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"run.py\", line 122, in run\n    model = train(model, config, experiment, training_directory, validation_directory, f'split_{split_index}')\n",
    "  File \"/home/steven/Github/keras-training-system/src/train.py\", line 127, in train\n    callbacks=callbacks)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/legacy/interfaces.py\", line 91, in wrapper\n    return func(*args, **kwargs)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/engine/training.py\", line 1415, in fit_generator\n    initial_epoch=initial_epoch)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/engine/training_generator.py\", line 213, in fit_generator\n    class_weight=class_weight)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/engine/training.py\", line 1215, in train_on_batch\n    outputs = self.train_function(ins)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py\", line 2666, in __call__\n    return self._call(inputs)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py\", line 2636, in _call\n    fetched = self._callable_fn(*array_vals)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/tensorflow/python/client/session.py\", line 1382, in __call__\n    run_metadata_ptr)\n",
    "  File \"/home/steven/anaconda3/envs/py36/lib/python3.6/site-packages/tensorflow/python/framework/errors_impl.py\", line 519, in __exit__\n    c_api.TF_GetCode(self.status.status))\n",
    "tensorflow.python.framework.errors_impl.InternalError: cuDNN Backward Filter function launch failure : input shape([16,128,149,149]) filter shape([3,3,128,128])\n\t [[Node: training/Nadam/gradients/block2_conv2/convolution_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=[\"loc:@training/Nadam/gradients/block2_conv2/convolution_grad/Conv2DBackpropInput\"], data_format=\"NCHW\", dilations=[1, 1, 1, 1], padding=\"SAME\", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device=\"/job:localhost/replica:0/task:0/device:GPU:0\"](block2_conv1/Relu, ConstantFolding/training/Nadam/gradients/block2_conv2/convolution_grad/ShapeN-matshapes-1, training/Nadam/gradients/block2_conv2/Relu_grad/ReluGrad)]]\n"
  ],
  "heartbeat": "2018-09-05T12:45:17.113873",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i7-2600 CPU @ 3.40GHz",
    "gpus": {
      "driver_version": "390.87",
      "gpus": [
        {
          "model": "GeForce GTX 980",
          "persistence_mode": false,
          "total_memory": 4041
        }
      ]
    },
    "hostname": "steven-Aurora-R3",
    "os": [
      "Linux",
      "Linux-4.15.0-33-generic-x86_64-with-debian-buster-sid"
    ],
    "python_version": "3.6.5"
  },
  "meta": {
    "command": "run",
    "options": {
      "--beat_interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print_config": false,
      "--priority": null,
      "--queue": false,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [],
      "help": false,
      "with": false
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2018-09-05T12:42:54.823440",
  "status": "FAILED",
  "stop_time": "2018-09-05T12:45:17.115606"
}